### PR TITLE
Remove ComproPago

### PR DESCRIPTION
**Pull request by adamrusted**
Contributes to #5251.

~Celery and~ ComproPago is ~are both~ displaying blank pages when I try to visit~them~.
Flagged here for removal, if they are no longer accessible.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1409,11 +1409,6 @@
             "hex": "DC0D15",
             "source": "https://www.cdprojekt.com/en/media/logotypes/"
         },
-        {
-            "title": "Celery",
-            "hex": "37814A",
-            "source": "http://www.celeryproject.org/"
```
Comment (service-paradis) [inline _data/simple-icons.json:1415]: The docs is still available:
https://docs.celeryproject.org/en/stable/

Comment (PeterShaggyNoble) [reply _data/simple-icons.json:1415]: Was about to say the same. Their Alexa rank is still within our scope and [their primary repo](https://github.com/celery/celery/) is still very much active, too.

Comment (adamrusted) [reply _data/simple-icons.json:1415]: Just spotted that. Went by the quick glance at the website - as that's all the info we have in the JSON. Will remove that one now.

Comment (service-paradis) [reply _data/simple-icons.json]: The source will need to be updated, for sure :)

**Comment by PeterShaggyNoble** [comment]
The blank page on ComproPago looks to be a JS issue but as it's Alexa rank is [pushing 1m](https://www.alexa.com/siteinfo/compropago.com) and [their GH repos] have no recent activity and no stars to speak of, it's definitely outside our scope.

**Comment by service-paradis** [comment]
> The blank page on ComproPago looks to be a JS issue but as it's Alexa rank is [pushing 1m](https://www.alexa.com/siteinfo/compropago.com) and [their GH repos] have no recent activity and no stars to speak of, it's definitely outside our scope.

I agree! No activity on their social media since 2018 too.

File: _data/simple-icons.json (hunk 1 above)
Comment (adamrusted) [reply _data/simple-icons.json]: Resolved in #5273, didn't want to mix updates / deletions in same PR.

**Review by PeterShaggyNoble — APPROVED** [review]
No changes since and the Alexa rank continues to fall so I think we're safe to remove this one. Thanks, @adamrusted 👍🏻

**Pull request merged**